lib/paperclip/storage/eitheror.rb

```diff
@@ -3,6 +3,7 @@ module Storage
     module Eitheror
       def self.extended base
         base.instance_eval do
+          base.options[:either][:enabled] = true if base.options[:either][:enabled].nil?
```

Comment:
I did but unfortunately, if we define `options[:either][:enabled] = false` in the model, that will still set it to true :/

My workaround was to explicitly check for `nil`.